config/spring.rb

```diff
@@ -4,3 +4,10 @@
   tmp/restart.txt
   tmp/caching-dev.txt
 ].each { |path| Spring.watch(path) }
+Spring.after_fork do
+  ENV['DEBUGGER_STORED_RUBYLIB']&.split(File::PATH_SEPARATOR)&.each do |path|
+    next unless path =~ /ruby-debug-ide/
+
+    load "#{path}/ruby-debug-ide/multiprocess/starter.rb"
+  end
+end
```

Comment:
Was this rubocop? Or was it something else you're doing?

Reply:
This is due to me using RubyMine as my IDE, can be removed but won't have an impact on live